### PR TITLE
Cleanup MONITORINFOEX

### PR DESCRIPTION
**Pull request by hughbe**
## Proposed Changes
- Cleanup MONITORINFOEX

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/dotnet/winforms/pull/2367)

### CODE REVIEW

File: src/System.Windows.Forms/src/System/Windows/Forms/Screen.cs [hunk 1]
```diff
@@ -76,13 +75,15 @@ internal Screen(IntPtr monitor, IntPtr hdc)
             else
             {
                 // Multiple monitor system
-                var info = new NativeMethods.MONITORINFOEX();
-                SafeNativeMethods.GetMonitorInfo(new HandleRef(null, monitor), info);
-                bounds = Rectangle.FromLTRB(info.rcMonitor.left, info.rcMonitor.top, info.rcMonitor.right, info.rcMonitor.bottom);
-                primary = ((info.dwFlags & MONITORINFOF_PRIMARY) != 0);
+                var info = new User32.MONITORINFOEXW
+                {
+                    cbSize = (uint)Marshal.SizeOf<User32.MONITORINFOEXW>()
```
Comment (RussKie) [inline src/System.Windows.Forms/src/System/Windows/Forms/Screen.cs]: Should this be?
```suggestion
                    cbSize = (uint)sizeof(User32.MONITORINFOEXW)
```

Comment (hughbe) [reply src/System.Windows.Forms/src/System/Windows/Forms/Screen.cs]: done

Comment (RussKie) [reply src/System.Windows.Forms/src/System/Windows/Forms/Screen.cs]: It hasn't changed here 😉

Comment (hughbe) [reply src/System.Windows.Forms/src/System/Windows/Forms/Screen.cs]: whoops! didn't push

**Pull request merged**